DEVELOPMENT.md

```diff
@@ -126,3 +126,39 @@ On Fedora, this can be fixed by adding a `.conf` file to `/etc/sysctl.d/ increas
 fs.inotify.max_user_watches=2097152
 fs.inotify.max_user_instances=256
 ```
+
+### Error: can only create exec sessions on running containers: container state improper
+
+This may happen when the containers have stopped. This can be fixed by deleting the already created clusters and starting the script again.
+
+```bash
+sudo kind delete clusters us-east1 us-west1
+```
+
+### Failed to create kind cluster - "Reached target ._Multi-User System._|detected cgroup v1"
+
+This is happening probably due to [Docker Desktop 4.3.0+ using Cgroups v2](https://kind.sigs.k8s.io/docs/user/known-issues/#failure-to-create-cluster-with-docker-desktop-as-container-runtime).
+
+To fix this issue, please ensure that the `KIND_EXPERIMENTAL_PROVIDER=podman` is set properly. You should be able to see _enabling experimental podman provider_ while running kind commands or the scripts. If the issue remains even after this, please try to disable docker and try again.
+
+A final fix is to uninstall docker and use podman only.
+
+### Looping with "error: availableReplicas is not found"
+
+This issue is related to the Argo CD installation. It waits for the ingress controller to be ready before creating the ingress object for it. You can wait or if you don't need Argo CD you can pass `NO_ARGOCD=true` before running the script.
+
+The other solution is to increase `fs.inotify.max_user_instances`. This can be done either by the solution described above or by running the following command.
```

Comment:
I would put some explanation: it has been noticed that a common reason for the ingress controller not becoming ready is scarcity of file monitors.... this is why increasing `fs.inotify.max_user_instances` may help.